web/robot/list_reordering.py

```diff
@@ -0,0 +1,55 @@
+import sys
+import math
+import random
+
+class Point():
+    def __init__(self, x, y):
+        self.x = x;
+        self.y = y;
+
+    def __eq__(self, other):
+        return (self.x == other.x) & (self.y == other.y)
+
+
```

Comment:
A more verbose function name might be helpful here. Something like "find_closest_neighbor". Also note the function naming style for this repo is _lowercase_with_underscores_